src/test/kotlin/de/bund/digitalservice/useid/widget/WidgetControllerIntegrationTest.kt

```diff
@@ -65,7 +85,24 @@ class WidgetControllerIntegrationTest(
             .expectHeader()
             .valueEquals(
                 "Content-Security-Policy",
-                "some default value;frame-ancestors 'self';",
+                contentSecurityPolicy.getDefaultCSPHeaderValue(),
```

Comment:
If you use the methods of the `contentSecurityPolicy` component to validate the result, you are not testing those methods any longer since the production code uses the same. So if we mess up those, the tests will not break. Either add new tests for the methods of `contentSecurityPolicy` or restore the behaviour from before by comparing to the expected outcome (String values).

Reply:
You are totally right. Thanks for clarifying on this one 🙌